EKF/common.h

```diff
@@ -319,6 +324,11 @@ struct parameters {
 		tas_innov_gate = 5.0f;		
   		eas_noise = 1.4f;			
 
+  		// sysntetic sideslip fusion
+  		beta_innov_gate = 5.0f;		// [CHECK THIS VALUE]
```

Comment:
A description of the parameter including units is required here, not a reminder note